spec/requests/videos_feed_spec.rb

```diff
@@ -0,0 +1,29 @@
+require "rails_helper"
+
+RSpec.describe "Videos Feed", :type => :request do
```

Comment:
Can you remove the `:type` tag as we use `infer_spec_type_from_file_location!` so this isn't required. Thx.